sdf.md

```diff
@@ -193,16 +193,12 @@ Thing:
 Grouping:
 : An sdfThing or sdfObject, i.e., (directly or indirectly) a combination of Affordances.
```

Comment:
I was wondering if this entry should be moved right before the affordances entries (i.e., before `Property`) to have it closer to its appearances within the terminology section. It could also be moved right after the `Group` entry to make it easier to see that the two terms are distinct.

Reply:
This move would require a lot more to move, e.g., Affordance.
I think we'd need a more complete proposal.
I'd rather merge this PR now and think about more improvements from that base.

Reply:
That makes sense :) I'll mark this thread as resolved then.